Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-Suspicious_Hidden_User_Created_Test.yml

```diff
@@ -1,1116 +1,3103 @@
       key:
         simple: LocalUserPasswordStatus
       value:
         complex:
+          root: Core.ScriptResult.results.[0].command_output
           filters:
-          - - left:
-                iscontext: true
+          - - operator: containsGeneral
+              left:
                 value:
                   simple: Core.ScriptResult.results.[0].command_output
-              operator: containsGeneral
+                iscontext: true
               right:
                 value:
                   simple: Password expires
-          root: Core.ScriptResult.results.[0].command_output
     separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1098,
+          "y": 1286
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
     skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "30":
+    id: "30"
+    taskid: 85e71fec-a160-4cfe-aa82-648feea810b3
+    type: condition
     task:
-      brand: ''
-      description: Extracts and saves the PASSWORDEXPIRES value of the locally created
-        user from the results of the Powershell script execution.
-      id: debbbea1-ba6b-4627-8d3b-a22bcc475682
-      iscommand: false
-      name: Extract password expiration flag
-      scriptName: SetAndHandleEmpty
-      type: regular
+      id: 85e71fec-a160-4cfe-aa82-648feea810b3
       version: -1
-    taskid: debbbea1-ba6b-4627-8d3b-a22bcc475682
-    timertriggers: []
-    type: regular
-    view: "{\n  \"position\": {\n    \"x\": 1100,\n    \"y\": 1260\n  }\n}"
-  '3':
-    continueonerrortype: ''
-    id: '3'
-    ignoreworker: false
-    isautoswitchedtoquietmode: false
-    isoversize: false
+      name: Analyst review - disable suspicious user?
+      description: "We found suspicious indicators in this investigation. However, the activity could not be verified with the user/s.\nPlease review the findings and decide on the next steps:\n\n### User who created the suspicious hidden user:\n`${Core.OriginalAlert.raw_abioc.event.evtlog_subject_username}`\n\n---\n\n### Involved Host:\n`${alert.hostname}`\n\n---\n\n### Hidden user was added to a privileged group:\n**Group name**: `${HiddenUserAddedToGroup}`\n\n---\n\n### The Suspicious hidden user that created:\n`${Core.OriginalAlert.raw_abioc.event.action_evtlog_data_fields.TargetUserName}`\n\n---\n\n### Related Alerts\nRelated Script Engine Activity alerts found in the incident: `${.=val.foundIncidents && val.foundIncidents.length > 0 ? \"True\" : \"False\"}`\n\n---\n\n### Suspicious Command lines executed on the \n`${.=val.SuspiciousCommandLines || \"N/A\"}`\n\n---\n\n##Login events with the hidden user:\n**Login results number**: `${.=val.HiddenUserLogin || \"N/A\"}`\n\n---\n\n### Hidden user access to share folder/service install\n`${.=val.ShareAccessorServiceInstall || \"N/A\"}`\n\n---\n\n### RDP\n`${.=val.RDPSessionsOnHost || \"N/A\"}`\n\n\n"
```

Comment:
For some reason this markdown won't render properly. Maybe add space after the `##`
![image](https://github.com/user-attachments/assets/ead479bb-0c0d-4e4a-b18b-297e898dcde7)